### PR TITLE
SwiftUI Kit, on the tube!

### PR DESCRIPTION
**Pull request by majouji**
Screen Shot 2020-07-11 at 22 40 25

Bam!

Also:
- Removed extraneous asset catalogs (all targets share the same one now)
- Changed a few conditional statements to exclude tvOS instead of including all 3 other platforms (little cleaner)

### Known issues
- Certain views don’t scroll, like FontsGrouping. May need an explicit `ScrollView`

### CODE REVIEW

File: Shared/Groupings/ButtonsGroup.swift [hunk 1]
```diff
@@ -45,7 +45,6 @@ struct ButtonsGroup: View {
                             #endif
                         }
                         
-                        #if os(iOS)
```
Comment (Nef10) [inline Shared/Groupings/ButtonsGroup.swift:48]: This does no longer compile on macOS, as actionSheet is not available there

Comment (Nef10) [reply Shared/Groupings/ButtonsGroup.swift:48]: Screen Shot 2020-07-11 at 19 53 45

Comment (majouji) [reply Shared/Groupings/ButtonsGroup.swift:48]: Whoops, resolved now. Good catch, thank you!

**Comment by Nef10** [comment]
👍

**Comment by jordansinger** [comment]
Wonderful! We can address the known issues in a follow-up.

**Pull request merged**